graal/backends/core/analyzers/lizard.py

```diff
@@ -94,3 +77,52 @@ def analyze(self, **kwargs):
 
         result['funs'] = funs_data
         return result
+
+    def __analyze_repository(self, repository_path, details):
+        """
+        """
+        analysis_result = []
+
+        analysis = lizard.analyze(
+            paths=[repository_path],
+            threads=1,
+            exts=lizard.get_extensions([]),
+        )
+        file_analysis = typing.cast(typing.Iterator[lizard.FileInformation], analysis)
```

Comment:
Can you explain the meaning of this statement ?

Reply:
Sure.
I couldn't find an example on how to execute `lizard` at the directory level (tried --help) still, I was doing something incorrectly. I read in a discussion forum that the earlier versions wouldn't return a `FileInformation` iterator (and I found this could be a potential error). 
So I had to typecast by mapping every object in analysis to an iterator of type `lizard.FileInformation` hence using the above line.

We're better off without using the line as the return type was resolved long back and the error that I was encountering was due to not passing the `exts` parameter. [here](https://github.com/chaoss/grimoirelab-graal/pull/39#discussion-diff-296118919R89).

I forgot to remove the line after I figured out the mistake. Thanks for pointing it out. 
I'll be careful for now on. I'll update the PR.

Thanks :)

Reply:
Great, thanks. Please let me know when I can check the PR again